src/pymser/pymser.py

```diff
@@ -84,13 +86,13 @@ def batch_average_data(data, batch_size=1):
     if batch_size > 1:
         # Trucate the data to allow a closed batch.
         # Be aware that this will remove the last points to make a closed batch
-        truncated_data = data[:int(torch.floor(len(data) / batch_size) * batch_size)]
+        truncated_data = data[: int(len(data) / batch_size // 1 * batch_size)]
```

Comment:
This is only a float and not an array/tensor so basic arithmetic (`// 1`) should be sufficient instead of `floor`